features/core_min_docker_build_image.feature

```diff
@@ -62,7 +62,7 @@ Feature:  Build Container images with SUSE Manager. Basic image
   And I run "zypper -n --gpg-auto-import-keys ref" on "sle-minion"
   And I apply highstate on "sle-minion"
   Then I wait until "docker" service is up and running on "sle-minion"
-  # FIXME: We need a test for image store with credentials
```

Comment:
added in issue of spacewalk-testsuite https://github.com/SUSE/spacewalk-testsuite-base/issues/329